python/ray/services.py

```diff
@@ -93,6 +93,34 @@ def include_java_from_redis(redis_client):
     return redis_client.get("INCLUDE_JAVA") == b"1"
 
 
+def find_redis_address_or_die():
+    import psutil
```

Comment:
since `psutil` is not a hard dependency for Ray, can we add something like

```python
try:
    import psutil
except ImportError:
    # What time of exception should this be??????????????????????????
    raise Exception("In order to automatically detect the Ray cluster, you "
                    "must install psutil. You can do this with 'pip install psutil'. "
                    "If that doesn't work, try using Anaconda Python and doing "
                    "'conda install psutil'.")